htdocs/reception/list.php

```diff
@@ -166,6 +169,10 @@
 }
 
 if (empty($reshook)) {
+	$uploaddir = $conf->reception->dir_output.'/temp/massgeneration/'.$user->id;
+	$objectclass = 'Reception';
+	include DOL_DOCUMENT_ROOT.'/core/actions_massactions.inc.php';
```

Comment:
There is this test into this file
	if (empty($diroutputmassaction)) {
		dol_print_error(null, 'include of actions_massactions.inc.php is done but var $diroutputmassaction was not defined');
		exit;
	}

So why having remove the

diroutputmassaction = $conf->reception->dir_output.'/temp/massgeneration/'.$user->id;

Reply:
You're right. I did not noticed that `$diroutputmassaction` was used in actions_massactions.inc.php. I tested it with custom massaction (that was not `builddoc`) so I did not have that error.

`$uploaddir` could be the same as `$diroutputmassaction` ?

Reply:
$uploaddir is the root of module. So
$uploaddir = $conf->reception->dir_output;
$diroutputmassaction is a temporary sub-dir of this directory.